gateway/assets/script/bootstrap.js

```diff
@@ -96,11 +96,18 @@ app.controller("home", ['$scope', '$log', '$http', '$location', '$interval', '$f
                     linkElement.setAttribute('href', url);
                     linkElement.setAttribute("download", filename);
          
-                    var clickEvent = new MouseEvent("click", {
-                        "view": window,
-                        "bubbles": true,
-                        "cancelable": false
-                    });
+                    var clickEvent;
```

Comment:
@iyovcheva please could you explain the new change here?

What do the magic numbers / flags mean or do when combined? i.e.

`"click",true,false,window,0,0,0,0,0,false,false,false,false,0,null`

Reply:
@alexellis 

```
object.initMouseEvent (eventName, bubbles, cancelable, view, detail, screenX, screenY, clientX, clientY, ctrlKey, altKey, shiftKey, metaKey, button, relatedTarget);
```

will add a comment in code

Reply:
Addressed. Modification is on another line.